functions/inbound.js

```diff
@@ -129,4 +129,31 @@ module.exports = {
     res.status(200).send(`You'll get a call at ${volunteerFields.phoneNumber} from Bed-Stuy Strong shortly connecting you to ${recordFields.phoneNumber}`);
   }),
 
+  // NOTE that this endpoint is currently being used for vax support callbacks
+  callback_basic: functions.https.onRequest(async (req, res) => {
```

Comment:
because this isn't currently authenticated 😬 , and because we have the creds for the vax airtable in firebase now, can we make this endpoint specific to the vax table? i think for now the simplest way (to avoid having to add more schemas etc) would be to just add a required query parameter to act as an auth token- in this case we could use the base id and compare it to the base id in the firebase config

Reply:
yeah that makes sense! just renamed the functioned and added a check for the vax support base before calling.

Reply:
instead of comparing the query param to a known string (`'vaxSupport'`) let's compare it to the airtable base id (`functions.config().airtable.vax_support_base_id`)- that way it's basically a secret to anyone who doesnt have access to that airtable

Reply:
ah gotcha! fixed